deployment/HDI-Spark/Notebooks/deploy-sparkdotnet-repl.sh

```diff
@@ -0,0 +1,116 @@
+#!/bin/bash
+#
+# Install Livy and SparkMagic on Head node
+#
+
+if  [[ $HOSTNAME == hn* ]] ;
+then
+    # Update Livy Jars
+    sudo wget https://ruinliureplhdistorage.blob.core.windows.net/ruinliurepl/livy_jar.zip
+    sudo unzip livy_jar.zip
+
+    sudo cp -rf livy_jar/* /usr/hdp/current/livy2-server/
+
+    # Update SparkMagic
+    sudo wget https://ruinliureplhdistorage.blob.core.windows.net/ruinliurepl/sparkmagic.zip
+    sudo unzip sparkmagic.zip
+    
+    if [ ! -d "/usr/bin/anaconda/lib/python2.7/site-packages/sparkmagic/kernels/sparkdotnetkernel" ]
+    then
+        sudo mkdir /usr/bin/anaconda/lib/python2.7/site-packages/sparkmagic/kernels/sparkdotnetkernel
+    fi
+
+    sudo cp -f sparkmagic/config.json /home/spark/.sparkmagic/
+    sudo cp -rf sparkmagic/* /usr/bin/anaconda/lib/python2.7/site-packages/sparkmagic/
+
+    # install sparkdotnetkernel
+    sudo /usr/bin/anaconda/bin/jupyter-kernelspec install /usr/bin/anaconda/lib/python2.7/site-packages/sparkmagic/kernels/sparkdotnetkernel
+
+    # Remove the temporary livy jars and sparkmagic files.
+    sudo rm livy_jar.zip
+    sudo rm -rf livy_jar
+    sudo rm sparkmagic.zip
+    sudo rm -rf sparkmagic
+fi
+
+#
+# Install SparkDotnet and Worker on Worker node
+#
+
+if  [[ $HOSTNAME == wn* ]] ;
+then
+    set +e
```

Comment:
why not set this at the top of the script ?

Reply:
Moved